src/main/kotlin/org/kamiblue/client/module/modules/client/CommandConfig.kt

```diff
@@ -15,16 +19,25 @@ internal object CommandConfig : Module(
     showOnArray = false,
     alwaysEnabled = true
 ) {
-    val prefix = setting("Prefix", ";", { false })
-    val toggleMessages = setting("Toggle Messages", false)
+    var prefix by setting("Prefix", ";", { false })
```

Comment:
Why is this a var??

Reply:
So it can be changed with ;prefix

Reply:
That doesn't explain why it's a var

Reply:
It is delegate property not the actual setting object